v3/src/components/graph/adornments/movable-point/movable-point.tsx

```diff
@@ -51,17 +51,19 @@ export const MovablePoint = observer(function MovablePoint(props: IProps) {
     initX = xMax - (xMax - xMin) / 4,
     initY = yMax - (yMax - yMin) / 4
 
-  // get attributes for use in coordinates box
+  // get attributes for use in coordinates box and for determining when to reset the point
+  // to the initial position when the attributes have changed
   const allAttributes = dataConfig?.dataset?.attributes,
     xAttrId = dataConfig?.attributeID('x') || '',
     yAttrId = dataConfig?.attributeID('y') || '',
     xAttr = allAttributes?.find(attr => attr.id === xAttrId),
     yAttr = allAttributes?.find(attr => attr.id === yAttrId),
     xAttrName = xAttr?.name ?? '',
-    yAttrName = yAttr?.name ?? ''
+    yAttrName = yAttr?.name ?? '',
+    xAttrNameRef = useRef(xAttrName),
+    yAttrNameRef = useRef(yAttrName)
 
   const showCoordinates = useCallback((event: MouseEvent) => {
-    if (!model.points) return
```

Comment:
These `if (!model.points) return` lines were introduced while troubleshooting a problem I ran into in the Add Movable Point PR. They aren't necessary. I should have removed them after solving the problem, but forgot.